numpy/add_newdocs.py

```diff
@@ -335,6 +336,7 @@ def iter_add(x, y, out=None):
                 addop(it[0], it[1], out=it[2])
                 it.iternext()
 
+            it.close()
```

Comment:
Did you forget to implement the `close()` method?

Reply:
bad copy-paste from the nditer-close pull request, removed

Reply:
Did we decide to drop `close` after all? I searched back through the PR comments, and I thought we'd agreed on a dual-purpose solution.